// containers/book/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import * as R from 'ramda'
import {Link} from 'react-router-dom'

import {fetchBookById} from 'actions'
import {getBookById} from 'selectors'

class Book extends Component {
  componentDidMount() {
    const bookId = this.props.match.params.id
    this.props.fetchBookById(bookId)
  }

  renderField() {
    const {book} = this.props
    const columnField = R.compose(
      R.toPairs,
      R.pick([
        'name',
        'author',
        'pages'
      ])
    )(book)

    return columnField.map(([key, value]) => (
      <div className='column' key={key}>
        <div className='ab-details-title'>
          <p>{key}</p>
        </div>
        <div className='ab-details-info'>
          {value}
        </div>
      </div>
    ))
  }

  renderContent() {
    const {book} = this.props

    return (
      <div className='thumbnail'>
        <div className='row'>
          <div className='col-md-6'>
            <img
              className='img-thumbnail'
              src={book.image}
              alt={book.name}
            />
          </div>
          <div className='col-md-6'>
            {this.renderField()}
          </div>
        </div>
        <div className='caption-full'>
          <h4>{book.name}</h4>
          <p>{book.description}</p>
        </div>
      </div>
    )
  }

  renderSidebar() {
    const {book} = this.props

    return (
      <div>
        <div className='form-group'>
          <h1>{book.name}</h1>
        </div>
        <Link to='/' className='btn btn-info btn-block'>
          Click here to go back
        </Link>
      </div>
    )
  }

  render() {
    return (
      <div className='view-container'>
        <div className='container'>
          <div className='row'>
            <div className='col-md-9'>
              {this.props.book && this.renderContent()}
            </div>
            <div className='col-md-3'>
              {this.props.book && this.renderSidebar()}
            </div>
          </div>
        </div>
      </div>
    )
  }
}

const mapStateToProps = state => {
  return {
    book: getBookById(state, state.bookPage.id)
  }
}

const mapDispatchToProps = {
  fetchBookById,
}

export default connect(mapStateToProps, mapDispatchToProps)(Book)
