// api/mockBooks.js
export default [
    {
        id: '1',
        categoryId: '1',
        name: 'Good Detective',
        description:
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta',
        image: '/uploads/9780525535546.jpg',
        author: 'John McMahon',
        pages: '450',
    },
    {
        id: '2',
        categoryId: '1',
        name: 'The missings ones',
        description:
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta',
        image: '/uploads/33558545._SY475_.jpg',
        author: 'Patricia Gibney',
        pages: '320',
    },
    {
        id: '3',
        categoryId: '4',
        name: 'Mystic',
        description:
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta',
        image: '/uploads/51shzfFW70L._SX328_BO1,204,203,200_.jpg',
        author: 'Jason Denzel',
        pages: '770',
    },
    {
        id: '4',
        categoryId: '5',
        name: 'A Brief History of Time',
        description:
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta',
        image: '/uploads/best_science_books_Brief_History.jpg',
        author: 'Stephen Hawking',
        pages: '430',
    },
    {
        id: '5',
        categoryId: '3',
        name: 'After the last dance',
        description:
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta',
        image: '/uploads/9780751561159.jpg',
        author: 'Sarra Manning',
        pages: '876',
    },
    {
        id: '6',
        categoryId: '2',
        name: 'A-Z of horror',
        description:
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta',
        image: '/uploads/Prepare-to-Be-Scared-The-Best-Supernatural-Horror-Books-2.jpg',
        author: 'Iain Rob Wright',
        pages: '1200',
    }
]
